tool/tbot/main.go

```diff
@@ -83,7 +83,8 @@ func Run(args []string, stdout io.Writer) error {
 	versionCmd := app.Command("version", "Print the version of your tbot binary.")
 
 	startCmd := app.Command("start", "Starts the renewal bot, writing certificates to the data dir at a set interval.")
-	startCmd.Flag("auth-server", "Address of the Teleport Auth Server or Proxy Server.").Short('a').Envar(authServerEnvVar).StringVar(&cf.AuthServer)
+	startCmd.Flag("auth-server", "Address of the Teleport Auth Server. Prefer using --proxy-server where possible.").Short('a').Envar(authServerEnvVar).StringVar(&cf.AuthServer)
+	startCmd.Flag("proxy-server", "Address of the Teleport Proxy Server.").StringVar(&cf.ProxyServer)
```

Comment:
What do you think about adding env var support as well?
Someone might be using it for setting the auth-server and now sees that this is _deprecated_, but can't change it because it doesn't support Env Vars

Reply:
I'm never a huge fan of offering three ways of tuning a value - but it may make sense for me to add this here just for the sake of consistency. I'll add it.

Reply:
fb77d28b55